docker/proof-fri-gpu-compressor-gar/Dockerfile

```diff
@@ -0,0 +1,18 @@
+ARG PROOF_FRI_GPU_IMAGE
```

Comment:
Let's call it `PROOF_COMPRESSOR_IMAGE`, that'll be more self-explanatory. Applies to other places as well